docs: Adding Doug Parker to contributors list.

dgp1130 opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

I am not in the contributors list.

Issue Number: N/A


## What is the new behavior?

I am now in the contributors list.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

I listed Igor as my lead, though I'm honestly not sure what that means, so let me know if I should use someone else.

dgp1130 committed:
docs: Adding Doug Parker to contributors list.

StephenFluin reviewed (changes requested):
@dgp1130 I think you combined two unrelated changes into this PR. Can you revert the docs change so and submit it as a different PR so we can approve them independently?

StephenFluin commented:
Also, if you can shrink the photo to around ~20kb, that would be great.

dgp1130 commented:
Just reverted it, apologies about that. I made a new branch but then forgot to switch to it before making the changes. 🤦‍♂

gkalpak commented on aio/content/marketing/contributors.json:
```diff
@@ -823,5 +823,14 @@
+    "picture": "dgp1130.jpg",
+    "twitter": "develwoutacause",
+    "website": "https://github.com/dgp1130",
+    "bio": "Doug is an overly-opionated software developer with a passion for making awesome developer tools. He is motivated primarily by his selfish desire to make his own life easier, but loves to share his tools and workflows with others to make everyone's lives easier.",
```
> overly-opionated

Did you, by any chance, mean _"overly-op**in**ionated"_? :grin:

dgp1130 replied:
Done, thanks for catching that.

gkalpak reviewed (commented):
Can you please remove the unrelated commits (`git revert` is not enough, because they will still show up in the main branches git history - you have to entirely remove the commits).

dgp1130 commented:
Reduced the file size of the photo, dropped the two unrelated commits, and squashed them all together. I've used Git in a solo manner before, but am still learning how PRs work with it.
